protocols/raft/src/main/java/io/atomix/protocols/raft/storage/RaftStorage.java

```diff
@@ -276,25 +280,15 @@ public void deleteMetaStore() {
   /**
    * Opens a new {@link SnapshotStore}, recovering snapshots from disk if they exist.
    *
-   * <p>The snapshot store will be loaded using based on the configured {@link StorageLevel}. If the
-   * storage level is persistent then the snapshot store will be loaded from disk, otherwise a new
-   * snapshot store will be created.
-   *
    * @return The snapshot store.
    */
   public SnapshotStore openSnapshotStore() {
```

Comment:
You're right; I changed the behaviour because before deleting the snapshot store was hardcoded as listing all `*.snapshot` files in the folder, and I wanted deletion to be specific to the implementation, so I had to keep the store instance around. That also explains the weird bidirectional dependency between `RaftStorage` and `SnapshotStore` which wasn't present before (i.e. it was just `SnapshotStore` -> `RaftStorage`).